frappe/database/database.py

```diff
@@ -94,6 +94,11 @@ def __init__(
 		self.db_name = frappe.conf.db_name
 		self._conn = None
 
+		self.sslmode = frappe.conf.db_ssl_mode or None
+		self.sslrootcert = frappe.conf.db_ssl_ca or None
+		self.sslcert = frappe.conf.db_ssl_cert or None
+		self.sslkey = frappe.conf.db_ssl_key or None
```

Comment:
these are specific to postgres - let's directly use the `frappe.conf.db.x` instead of defining them again in `postgres/database.py`?

mariadb already has these implemented:
https://github.com/frappe/frappe/blob/de490fc1f7322c8ed0de2bc2924387f27d86c740/frappe/database/mariadb/database.py#L131-L136

Reply:
also, do we need the `db_ssl_mode`? it's by default `prefer` i.e 

> first try an SSL connection; if that fails, try a non-SSL connection

ref: https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS